app/js/Editor.js

```diff
@@ -249,7 +251,7 @@ class Editor extends EventEmitter {
       fs.writeFileSync(this.tmp, JSON.stringify(this.data))
```

Comment:
We could prob use the same `tmpRelative` path here. Let's change it to `tmpDir` though if we delete `this.tmp`.